homeassistant/components/rfxtrx/__init__.py

```diff
@@ -466,7 +467,9 @@ def __init__(
         event: rfxtrxmod.RFXtrxEvent | None = None,
     ) -> None:
         """Initialize the device."""
-        self._name = f"{device.type_string} {device.id_string}"
+        self._attr_name = None
+        self._attr_assumed_state = True
+        self._attr_should_poll = False
```

Comment:
Could `assumed_state` and `should_poll` be class attributes? (not instance attributes)

Reply:
Yup

Reply:
Extra note: I think there are quite a few properties in `rfxtrx` that could be migrated to instance or class attributes, for example the `unique_id`. I think you should consider it for a follow-up PR.